### PR TITLE
Page inscription

### PR DESCRIPTION
**Pull request by Marie-Garde**

### CODE REVIEW

**Review by Gauthier-Lhomme — DISMISSED** [review]

**Review by Thoms-C — DISMISSED** [review]
Ok!

**Comment by LoicBrassart** [comment]
Please solve conflicts before assigning me ;-)

File: src/components/Connection.jsx [hunk 1]
```diff
@@ -0,0 +1,61 @@
...
+                name="password"
+                id="password"
+                placeholder="Mot de passe"
+                required
+              />
+            </DivLabel>
+            <Button>Confirmer</Button>
+          </Formulaire>
+          <H2>
+            Pas encore de compte ?{" "}
```
Comment (LoicBrassart) [inline src/components/Connection.jsx]: Remove {" "}

File: src/styled-components/Connection.jsx [hunk 2]
```diff
@@ -0,0 +1,163 @@
...
+  font-size: 0.8rem;
+  font-family: "Noto Sans CJK JP";
+  @media ${Device.mobile} {
+    font-size: 0.9rem;
+    margin-bottom: 2vh;
+    margin-top: 2vh;
+  }
+`;
+
+//questionnaire
```
Comment (LoicBrassart) [inline src/styled-components/Connection.jsx]: Comments should be in English

File: src/styled-components/SignUp.jsx [hunk 3]
```diff
@@ -0,0 +1,158 @@
...
+  font-size: 0.8rem;
+  font-family: "Noto Sans CJK JP";
+  @media ${Device.mobile} {
+    font-size: 0.9rem;
+    margin-bottom: 2vh;
+    margin-top: 2vh;
+  }
+`;
+
+//questionnaire
```
Comment (LoicBrassart) [inline src/styled-components/SignUp.jsx]: Comments should be in English

**Review by LoicBrassart — CHANGES_REQUESTED** [review]
Some fixes to do

**Review by Gauthier-Lhomme — APPROVED** [review]

**Review by LoicBrassart — APPROVED** [review]
NmE

**Pull request merged**